test/sharness/t0087-repo-robust-gc.sh

```diff
@@ -74,8 +78,8 @@ test_gc_robust_part1() {
   test_expect_success "'ipfs repo gc' should still run and remove as much as possible" '
     test_must_fail ipfs repo gc 2>&1 | tee repo_gc_out &&
     grep -q "could not remove $LEAF2" repo_gc_out &&
-    grep -q "removed $LEAF3" repo_gc_out &&
-    grep -q "removed $LEAF4" repo_gc_out &&
+    grep -q "removed $(to_raw_cid $LEAF3)" repo_gc_out &&
+    grep -q "removed $(to_raw_cid $LEAF4)" repo_gc_out &&
```

Comment:
These don't seem necessary since we're already doing raw leaves, but it can't hurt